src/model_outputfiles.cpp

```diff
@@ -18,7 +18,9 @@ void Model::createReport(){
 }
 
 void Model::createReport(std::string path){
+  const double avogadro = 6.02214076e23;

```

Comment:
I think it would be good to write constants like these, that should never ever be changed, in all-caps. I've seen this stylistic convention a few times now